### ISSUE TITLE
[Bug]: Xcode 15.3 and App Store Connect Issue 

### ISSUE THREAD

**Issue by gooson** [braze-inc/braze-swift-sdk#112]
### Platform

iOS

### Platform Version

iOS 17.4

### Braze SDK Version

8.1.0

### Xcode Version

Xcode 15.3

### Computer Processor

Apple (M1)

### Repro Rate

100%

### Steps To Reproduce

Archive app and upload it to the App Store.

<img width="1634" alt="스크린샷 2024-03-08 오전 11 21 34" src="https://github.com/braze-inc/braze-swift-sdk/assets/1947069/874f3cc2-b589-4f37-904b-63d8f584041f">

### Expected Behavior

Expect successful upload to App Store Connect

### Actual Incorrect Behavior

upload failed

Asset validation failed
Invalid Bundle. The bundle KREAM Inhouse.app/Frameworks/BrazeKit.framework does not support the minimum OS Version specified in the Info.plist. (ID: 6b5c0795-e051-4c78-b7b1-a0e52567645c)

Asset validation failed
Invalid Bundle. The bundle KREAM Inhouse.app/Frameworks/BrazeLocation.framework does not support the minimum OS Version specified in the Info.plist. (ID: aa07ed5a-007d-4fa0-a82e-8aacc98ef36e)

### Verbose Logs

```shell
Asset validation failed
Invalid Bundle. The bundle KREAM Inhouse.app/Frameworks/BrazeKit.framework does not support the minimum OS Version specified in the Info.plist. (ID: 6b5c0795-e051-4c78-b7b1-a0e52567645c)

Asset validation failed
Invalid Bundle. The bundle KREAM Inhouse.app/Frameworks/BrazeLocation.framework does not support the minimum OS Version specified in the Info.plist. (ID: aa07ed5a-007d-4fa0-a82e-8aacc98ef36e)
```

**Comment by jerielng** [comment]
Hi @gooson, thanks for sharing some additional details around this issue. We have a separate thread in #105, which I saw that you commented on. Although we believe this to be an Xcode bug, this is on our radar, and we'll continue to look into our options.

In the meantime, you can work around this issue by either using Xcode 15.2 or sourcing your Braze SDK from our [prebuilt dynamic XCFrameworks repository](https://github.com/braze-inc/braze-swift-sdk-prebuilt-dynamic).

We'll continue to keep you updated. Feel free to keep this issue open, or you may close it if you wish to continue following this topic in #105. Thanks!

**Comment by jesus-mg-ios** [comment]
Same issue here.

**Comment by jesus-mg-ios** [comment]
Seems like firebase had the same issue and solve it. https://github.com/firebase/firebase-ios-sdk/issues/12441

**Comment by stuti7** [comment]
Same issue here

**Comment by bstillitano** [comment]
This is breaking our builds also. Any ETA on fixing this?

**Comment by hokstuff** [comment]
We have made some changes in our private repo that are currently being validated. We will be in touch once it is verified and released! Until then, using Xcode 15.2 or using our [prebuilt dynamic xcframework](https://github.com/braze-inc/braze-swift-sdk/issues/112#issuecomment-1985769816) are valid workarounds.

Thanks!

**Comment by bstillitano** [comment]
What is the ETA on that @hokstuff? That answer seems very vague and raises concerns around how long we'll actually be waiting for this.

**Comment by hokstuff** [comment]
Hi @gooson @jesus-mg-ios @stuti7 @bstillitano,

We have just released [Braze Swift SDK version 8.2.1](https://github.com/braze-inc/braze-swift-sdk/releases/tag/8.2.1) which fixes the issues during archiving on Xcode 15.3.

Let us know if you run into any further issues - Thanks!

**Comment by bstillitano** [comment]
@hokstuff confirming this is working. We did receive an error initially after uploading to ASC saying `Uploading IPA failed: signal: segmentation fault` however a subsequent build worked. Will report back if any issues. Appreciate the hustle on this.
